deploy/clowdapp.yaml

```
@@ -20,6 +20,7 @@ objects:
       - content-sources-backend
       - sources-api
       - rhsm-api-proxy
+      - insights-results-smart-proxy
```

Updating to `ccx-smart-proxy`

```suggestion
      - ccx-smart-proxy
```